### PR TITLE
Updating the service url to platform.hmcts.net from .internal

### PR DESCRIPTION
**Pull request by remya-james**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] the `keep-helm` label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/FPET-419

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: charts/prl-cos/values.yaml [hunk 1]
```diff
@@ -1,41 +1,41 @@
 java:
   applicationPort: 4044
-  ingressHost: prl-cos-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
+  ingressHost: prl-cos-{{ .Values.global.environment }}.{{ .Values.global.environment }}.platform.hmcts.net
```
Comment (priteshkr) [inline charts/prl-cos/values.yaml:3]: Deprecated for AAT and Preview only not for prod.

https://hmcts-reform.slack.com/archives/CA4F2MAFR/p1675868743958669

File: src/main/resources/application.yaml [hunk 2]
```diff
@@ -424,4 +424,4 @@ courts:
 
 fis_hearing:
   api:
-    url: ${HEARING_API_BASEURL:http://fis-hmc-api-demo.service.core-compute-demo.internal}
+    url: ${HEARING_API_BASEURL:http://fis-hmc-api-demo.demo.platform.hmcts.net}
```
Comment (priteshkr) [inline src/main/resources/application.yaml:427]: Don't add it for demo URLS

File: src/functionalTest/resources/requests/bundle/CreateBundleRequest.json [hunk 3]
```diff
@@ -1 +1 @@
...
...ocuments/8ab34183-a942-40f4-9fed-eb304a838937/binary","document_filename":"C100FinalDocument.pdf","document_hash":null,"document_creation_date":null},"documentFileName":"C100FinalDocument.pdf","documentGroup":"applicantApplication"}},{"id":null,"value":{"documentLink":{"document_url":"http://dm-store-demo.demo.platform.hmcts.net/documents/6fb20836-3022-4d5b-896e-7eb619b792bd","document_binary_url":"http://dm-store-demo.demo.platform.hmcts.net/documents/6fb20836-3022-4d5b-896e-7eb619b792bd/binary","document_filename":"SampleDoc1.pdf","document_hash":null,"document_creation_date":null},"documentFileName":"SampleDoc1.pdf","documentGroup":"applicantMiamCertificate"}},{"id":null,"value":{"documentLink":{"document_url":"http://dm-store-demo.demo.platform.hmcts.net/documents/349cc681-f3b0-4580-bef5-f71048a72cc1","document_binary_url":"http://dm-store-demo.demo.platform.hmcts.net/documents/349cc681-f3b0-4580-bef5-f71048a72cc1/binary","document_filename":"SampleDoc2.pdf","document_hash":null,"document_creation_date":null},"documentFileName":"SampleDoc2.pdf","documentGroup":"applicantPreviousOrdersSubmittedWithApplication"}}]}},"id":"ApplicantFname ApplicantLname"},"event_id":"createBundle"}
```
Comment (priteshkr) [inline src/functionalTest/resources/requests/bundle/CreateBundleRequest.json:1]: What is the change made here

File: src/functionalTest/resources/responses/bundle/ValidResponseBody.json [hunk 4]
```diff
@@ -1 +1 @@
...
...ame":"Letters from School","documents":[]}},{"value":{"name":"Other Evidences","documents":[]}}]}},{"value":{"name":"Other Witness Statements","documents":[],"folders":null}}],"sortIndex":2}},{"value":{"name":"Expert Reports including Guardian (CAFCASS)","documents":[],"folders":[{"value":{"name":"CAFCASS Report and Guardian","documents":[],"folders":[{"value":{"name":"Other Documents","documents":[]}}]}},{"value":{"name":"Expert Reports","documents":[],"folders":[{"value":{"name":"Medical Reports","documents":[]}},{"value":{"name":"DNA Reports","documents":[]}},{"value":{"name":"Medical Records","documents":[]}},{"value":{"name":"Drug And Alcohol Tests(Toxicology)","documents":[]}},{"value":{"name":"Police Reports","documents":[]}},{"value":{"name":"Other Reports","documents":[]}}]}}],"sortIndex":3}}],"stitchStatus":"NEW","stitchedDocument":null,"hasCoversheets":"No","hasTableOfContents":"Yes","hasFolderCoversheets":"Yes","stitchingFailureMessage":null,"fileName":"Applicant Firstname Applicant Lastname-Bundle.pdf","paginationStyle":"bottomRight","coverpageTemplate":"FL-PRL-LET-ENG-CoverPage.docx","pageNumberFormat":"pageRange"}}]},"errors":[],"warnings":[],"documentTaskId":180790}
```
Comment (priteshkr) [inline src/functionalTest/resources/responses/bundle/ValidResponseBody.json:1]: Why this change is required here

File: charts/prl-cos/values.yaml (hunk 1 above)
Comment (remya-james) [reply charts/prl-cos/values.yaml]: reverted for prod

File: charts/prl-cos/values.yaml [hunk 5]
```diff
@@ -1,41 +1,41 @@
...
   aadIdentityName: prl
   autoscaling:
     enabled: true
     maxReplicas: 9
     minReplicas: 6
     targetCPUUtilizationPercentage: 80 # Default is 80% target CPU utilization
   environment:
-    PRD_API_BASEURL: http://rd-professional-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
+    PRD_API_BASEURL: http://rd-professional-api-{{ .Values.global.environment }}.{{ .Values.global.environment }}.platform.hmcts.net
```
[the remark this answers is not in the document]
Comment (remya-james) [reply charts/prl-cos/values.yaml]: done

File: src/main/resources/application.yaml (hunk 2 above)
Comment (remya-james) [reply src/main/resources/application.yaml]: reverted

File: src/functionalTest/resources/responses/bundle/ValidResponseBody.json (hunk 4 above)
Comment (remya-james) [reply src/functionalTest/resources/responses/bundle/ValidResponseBody.json]: reverted for demo

File: src/functionalTest/resources/requests/bundle/CreateBundleRequest.json (hunk 3 above)
Comment (remya-james) [reply src/functionalTest/resources/requests/bundle/CreateBundleRequest.json]: reverted for demo
